x-pack/plugins/siem/public/components/stat_items/index.tsx

```diff
@@ -49,11 +52,69 @@ export interface StatItems {
 }
 
 export interface StatItemsProps extends StatItems {
-  key: string;
   areaChart?: AreaChartData[];
   barChart?: BarChartData[];
 }
 
+export const useKpiMatrixStatus = (
```

Comment:
nit - might be interesting to see a unit test just for your hook.

Reply:
Good point! Added it here:
https://github.com/elastic/kibana/blob/405f2eefbd37d504f41b1af9939a9ed1b1df1f82/x-pack/plugins/siem/public/components/stat_items/index.test.tsx#L186-L225